packages/cli/src/commands/virtualdevices/create.ts

```diff
@@ -0,0 +1,111 @@
+import { Flags } from '@oclif/core'
+import {
+	Device,
+	VirtualDeviceCreateRequest,
+} from '@smartthings/core-sdk'
+import {
+	APIOrganizationCommand,
+	InferredFlagsType,
+	inputAndOutputItem,
+	userInputProcessor,
+} from '@smartthings/cli-lib'
+import { buildTableOutput } from '../../lib/commands/devices/devices-util'
+
+import { chooseLocation } from '../locations'
+import { chooseDeviceName, chooseDeviceProfileDefinition } from '../../lib/commands/virtualdevices/virtualdevices-util'
+import {chooseRoom} from '../../lib/commands/locations/rooms/rooms-util'
+
+
+export default class VirtualDeviceCreateCommand extends APIOrganizationCommand<typeof VirtualDeviceCreateCommand.flags> {
```

Comment:
Yes, because it is the only one with an option to select a device profile from a list, and the list of device profiles is dependent on the organization.